s3/sc-portfolio-s3-basic.yaml

```diff
@@ -66,76 +22,34 @@ Parameters:
     Type: String
     Description: Last updated date and time of portfolio
     Default: ''
-Conditions:
-  CreateLaunchConstraint: !Equals
-    - !Ref 'LaunchRoleName'
-    - ''
-  CondPrincipalRoleName1: !Not
-    - !Equals
-      - !Ref 'PrincipalRoleName1'
-      - ''
-  CondPrincipalGroupName1: !Not
-    - !Equals
-      - !Ref 'PrincipalGroupName1'
-      - ''
-  CondPrincipalRoleName2: !Not
-    - !Equals
-      - !Ref 'PrincipalRoleName2'
-      - ''
-  CondPrincipalGroupName2: !Not
-    - !Equals
-      - !Ref 'PrincipalGroupName2'
-      - ''
 Resources:
   SCS3portfolio:
     Type: AWS::ServiceCatalog::Portfolio
     Properties:
       ProviderName: !Ref 'PortfolioProvider'
       Description: !Ref 'PorfolioDescription'
       DisplayName: !Ref 'PorfolioName'
-  LaunchConstraintRole:
-    Type: AWS::CloudFormation::Stack
-    Condition: CreateLaunchConstraint
-    Properties:
-      TemplateURL: !Sub '${RepoRootURL}iam/sc-s3-launchrole.yaml'
-      TimeoutInMinutes: 5
   LinkEndusersRole:
     Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
-    Condition: CondPrincipalRoleName1
     Properties:
-      PrincipalARN: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${PrincipalRoleName1}'
+      PrincipalARN: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-enduser-iam-ServiceCatalogEndusers-RoleArn'
       PortfolioId: !Ref 'SCS3portfolio'
       PrincipalType: IAM
   LinkEndusersGroup:
     Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
-    Condition: CondPrincipalGroupName1
```

Comment:
Consider leaving this condition too, which would give us the option of grouping roles. Maybe we want a Synapse team to map to a group?